extensions/amp-story/1.0/amp-story.js

```diff
@@ -719,102 +710,6 @@ export class AmpStory extends AMP.BaseElement {
     return true;
   }
 
-
-  /** @private */
-  isAmpStoryEnabled_() {
-    const {win} = this;
-    const {location} = win;
-
-    if (isExperimentOn(win, 'amp-story-v1') || getMode().test ||
-        location.protocol === 'file:') {
-      return true;
-    }
-
-    const origin = Services.urlForDoc(this.element).getSourceOrigin(location);
-    return this.isOriginWhitelisted_(origin);
-  }
-
-
-  /**
-   * @param {string} domain The domain part of the origin, to be hashed.
-   * @return {string} The hashed origin.
-   * @private
-   */
-  hashOrigin_(domain) {
-    return stringHash32(domain.toLowerCase());
-  }
-
-
-  /**
-   * @param {string} origin The origin to check.
-   * @return {boolean} Whether the specified origin is whitelisted to use the
-   *     amp-story extension.
-   * @private
-   */
-  isOriginWhitelisted_(origin) {
-    const {hostname} = Services.urlForDoc(this.element).parse(origin);
-    const domains = hostname.split('.');
-
-    // Check all permutations of the domain to see if any level of the domain is
-    // whitelisted.  Taking the example of the whitelisted domain
-    // example.co.uk, if the page is served from www.example.co.uk/page.html:
-    //
-    //   www.example.co.uk => false
-    //   example.co.uk => true
-    //   co.uk => false
-    //   uk => false
-    //
-    // This is necessary, since we don't have any guarantees of which level of
-    // the domain is whitelisted.  For many domains (e.g. .com), the second
-    // level of the domain is likely to be whitelisted, whereas for others
-    // (e.g. .co.uk) the third level may be whitelisted.  Additionally, this
-    // allows subdomains to be whitelisted individually.
-    return domains.some((unusedDomain, index) => {
-      const domain = domains.slice(index, domains.length).join('.');
-      const domainHash = this.hashOrigin_(domain);
-      return this.originWhitelist_.includes(domainHash);
-    });
-  }
-
-
-  /** @private */
-  assertAmpStoryExperiment_() {
-    if (this.isAmpStoryEnabled_()) {
-      return;
-    }
-
-    const errorIconEl = this.win.document.createElement('div');
-    errorIconEl.classList.add('i-amphtml-story-experiment-icon');
-    errorIconEl.classList.add('i-amphtml-story-experiment-icon-error');
-
-    const errorMsgEl = this.win.document.createElement('span');
-    errorMsgEl.textContent = this.localizationService_.getLocalizedString(
-        LocalizedStringId.AMP_STORY_WARNING_EXPERIMENT_DISABLED_TEXT);
-
-    const experimentsLinkEl = this.win.document.createElement('button');
-    experimentsLinkEl.textContent = this.localizationService_
-        .getLocalizedString(
-            LocalizedStringId.AMP_STORY_EXPERIMENT_ENABLE_BUTTON_LABEL);
```

Comment:
Sorry for the back and forth, but could you please deprecate the `LocalizedStringId`?

Reply:
Thanks for catching this!  No need to apologize 😃